packages/integration-react/src/router/initialize.ts

```diff
@@ -0,0 +1,32 @@
+import type { Agent } from '@grafana/agent-web';
+
+import type { ReactIntegrationConfig } from '../types';
+import { ReactRouterVersion } from './types';
+import { initializeReactRouterV4V5Instrumentation } from './v4v5';
+import { initializeReactRouterV6Instrumentation } from './v6';
+
+export function initializeReactRouterInstrumentation(config: ReactIntegrationConfig, agent: Agent): void {
+  if (!agent.api.isOTELInitialized()) {
+    agent.internalLogger.error(
+      'The Grafana React Router instrumentation requires tracing instrumentation. Please enable it in the "instrumentations" section of your config.'
```

Comment:
it's also possible to config otel separatetly and then call `agent.api.initOTEL`

Reply:
even if no tracing, we can (and should?) use `agent.api.pushEvent` to create a `navigation` event?

Reply:
Yeah. Right now it uses the events API :D. Gonna remove this check.